src/game/fasttext.c

```diff
@@ -11,7 +11,6 @@
 __asm__(
  ".section \".rodata\", \"a\", @progbits\n"
  ".balign 16\n"
```

Comment:
Wat (also please don't touch this, that's just another merge conflict for later when merging 2.4)

Reply:
fast_text would get added as a symbol when testing, though with a new stack heuristic maybe that's not the case anymore; will test

Reply:
No longer a concern since the 2.4 rebase